### ISSUE TITLE
Logos

### ISSUE THREAD

**Issue by carlstratton** [hutosaha/hrns-#98]
Hi guys, see the logos, attached. I've used PNGs for now. We should use the grey for the agencies without a photo upload.
[Archive.zip](https://github.com/hutosaha/hrns-/files/235437/Archive.zip)

**Comment by hdrdavies** [comment]
Thanks @carlstratton !

**Comment by hdrdavies** [comment]
![harness-logo-blue](https://cloud.githubusercontent.com/assets/13098199/14913436/fc4e3a30-0dfa-11e6-81d0-f10a23728c48.png)
![harness-logo-white](https://cloud.githubusercontent.com/assets/13098199/14913438/fed406cc-0dfa-11e6-9a90-274a2f64c187.png)
![h-icon-300x300](https://cloud.githubusercontent.com/assets/13098199/14913441/0a50d746-0dfb-11e6-9f18-695603b8d284.png)

**Comment by hdrdavies** [comment]
@carlstratton we don't have adobe illustrator - What are the .ai files?

**Comment by carlstratton** [comment]
The ai files are for Adobe Illustrator – Did I not send over the PNG files?
Apologies if not, i'll resend this evening, not sure what happened there.


Carl


*— *

t: 07824 632 199
e: cgstratton@gmail.com

senior UX designer at farfetch <http://www.farfetch.com>
co-founder | emblzn.com
designer | carlstratton.co.uk

**Comment by carlstratton** [comment]
The files you just added to the issue are PNG files, you can use them

Carl


*— *

t: 07824 632 199
e: cgstratton@gmail.com

senior UX designer at farfetch <http://www.farfetch.com>
co-founder | emblzn.com
designer | carlstratton.co.uk

**Comment by hdrdavies** [comment]
@carlstratton Yep Carl, sorry for the confusion - we've got the png files so no need to worry. Did you manage to get your Linkedin ID so I can make you an admin? Log in as a candidate from the homepage at hrns.herokuapp.com and it'll be at the top of the page

same for you @joshbrierley

**Comment by carlstratton** [comment]
Think this is my ID: AAIAAAPJf2wBTxKAubuuEpu3xTwHCYnk3UB_cbw


Carl


*— *

t: 07824 632 199
e: cgstratton@gmail.com

senior UX designer at farfetch <http://www.farfetch.com>
co-founder | emblzn.com
designer | carlstratton.co.uk

**Comment by hdrdavies** [comment]
Are you sure? I think it's only 10 characters long. Should come up like this:

![image](https://cloud.githubusercontent.com/assets/13098199/14917227/ea54851c-0e15-11e6-976a-aefc626e5107.png)

**Comment by carlstratton** [comment]
I'm definitely not sure! How do you find it?

Carl


*— *

t: 07824 632 199
e: cgstratton@gmail.com

senior UX designer at farfetch <http://www.farfetch.com>
co-founder | emblzn.com
designer | carlstratton.co.uk

**Comment by hdrdavies** [comment]
Log in as a candidate from the home page @carlstratton

**Comment by joshbrierley** [comment]
Sorry for the delay Huw, I've logged in as candidate is that all you needed me to do?

Josh Brierley
Consultant

T:  02037523043 M: 07770973292 E:Josh.Brierley@SailixSearch.com<mailto:ben.brierley@sailixsearch.com>
W: www.sailix.co.uk<http://www.sailix.co.uk/>  A: Sailix Ltd. 3rd Floor, 207 Regent St. London W1B 3HH.

TWITTER<https://twitter.com/SailixUK>   FACEBOOK<https://www.facebook.com/sailix> LINKEDIN<https://www.linkedin.com/company/9999596?trk=tyah&trkInfo=clickedVertical%3Acompany%2Cidx%3A1-1-1%2CtarId%3A1435564127004%2Ctas%3Asailix>

TECHNOLOGY TALENT. DIGITALLY REMASTERED.
[SAILIX VIBE]

Confidentiality Notice: The information contained in this email and any attachments is confidential. It is intended only for the named addressee(s). If  you are not the named addressee, please notify the sender immediately and do not disclose, copy or distribute the contents to any person other than the intended addressee(s). Company Registration Number: 9465157 Sailix Ltd, 2 Kelney, Melksham. Wilts. SN10 6ET.
________________________________

**Comment by hdrdavies** [comment]
@joshbrierley you need to send me the ID which is printed out at the top of the page, as shown in the screenshot. You can just comment it below.

**Comment by joshbrierley** [comment]
Can you send me the link again?


Josh Brierley
Consultant

T:  02037523043 M: 07770973292 E:Josh.Brierley@SailixSearch.com<mailto:ben.brierley@sailixsearch.com>
W: www.sailix.co.uk<http://www.sailix.co.uk/>  A: Sailix Ltd. 3rd Floor, 207 Regent St. London W1B 3HH.

TWITTER<https://twitter.com/SailixUK>   FACEBOOK<https://www.facebook.com/sailix> LINKEDIN<https://www.linkedin.com/company/9999596?trk=tyah&trkInfo=clickedVertical%3Acompany%2Cidx%3A1-1-1%2CtarId%3A1435564127004%2Ctas%3Asailix>

TECHNOLOGY TALENT. DIGITALLY REMASTERED.
[SAILIX VIBE]

Confidentiality Notice: The information contained in this email and any attachments is confidential. It is intended only for the named addressee(s). If  you are not the named addressee, please notify the sender immediately and do not disclose, copy or distribute the contents to any person other than the intended addressee(s). Company Registration Number: 9465157 Sailix Ltd, 2 Kelney, Melksham. Wilts. SN10 6ET.
________________________________

**Comment by hdrdavies** [comment]
http://hrns.herokuapp.com/login/candidate

**Comment by joshbrierley** [comment]
id: mXqGGl5IR1



Josh Brierley
Consultant

T:  02037523043 M: 07770973292 E:Josh.Brierley@SailixSearch.com<mailto:ben.brierley@sailixsearch.com>
W: www.sailix.co.uk<http://www.sailix.co.uk/>  A: Sailix Ltd. 3rd Floor, 207 Regent St. London W1B 3HH.

TWITTER<https://twitter.com/SailixUK>   FACEBOOK<https://www.facebook.com/sailix> LINKEDIN<https://www.linkedin.com/company/9999596?trk=tyah&trkInfo=clickedVertical%3Acompany%2Cidx%3A1-1-1%2CtarId%3A1435564127004%2Ctas%3Asailix>

TECHNOLOGY TALENT. DIGITALLY REMASTERED.
[SAILIX VIBE]

Confidentiality Notice: The information contained in this email and any attachments is confidential. It is intended only for the named addressee(s). If  you are not the named addressee, please notify the sender immediately and do not disclose, copy or distribute the contents to any person other than the intended addressee(s). Company Registration Number: 9465157 Sailix Ltd, 2 Kelney, Melksham. Wilts. SN10 6ET.
________________________________

**Comment by tormod17** [comment]
@carlstratton  can you send me the black logo you guys want instead, thanks
